client/components/domains/use-my-domain/domain-input.jsx

```diff
@@ -54,21 +54,18 @@ function UseMyDomainInput( {
 		? __( 'Enter the domain you would like to use:' )
 		: __( 'Enter your domain here' );
 
-	const hasDomainNoteLabel =
-		[ 'en', 'en-gb' ].includes( locale ) ||
-		hasTranslation(
-			'Please enter your site address without “www” or “https://” e.g. mydomain.com'
-		);
-	const domainNote = hasDomainNoteLabel
-		? __( 'Please enter your site address without “www” or “https://” e.g. mydomain.com' )
-		: '';
-
 	const hasDomainInputNoteLabel =
 		[ 'en', 'en-gb' ].includes( locale ) ||
 		hasTranslation( 'Enter the domain you would like to use:' );
 	const domainInputNote = hasDomainInputNoteLabel
 		? __( 'This won’t affect your existing site.' )
 		: '';
+
+	const hasDomainPlaceholderLabel =
+		[ 'en', 'en-gb' ].includes( locale ) || hasTranslation( 'yoursiteaddress.com' );
```

Comment:
FYI there's also variable for these but not a biggie as-is. 

https://github.com/Automattic/wp-calypso/blob/677fe5423fc170082be56e54b5ca3c2c009a7774/packages/i18n-utils/src/locales.ts#L25